clients/drcachesim/tests/view_test.cpp

```diff
@@ -607,25 +611,49 @@ run_regdeps_test(void *drcontext)
         { TRACE_TYPE_THREAD, 0, { t1 } },
         { TRACE_TYPE_PID, 0, { t1 } },
         { TRACE_TYPE_MARKER, TRACE_MARKER_TYPE_CACHE_LINE_SIZE, { 64 } },
-        { TRACE_TYPE_MARKER, TRACE_MARKER_TYPE_CHUNK_INSTR_COUNT, { 2 } },
+        { TRACE_TYPE_MARKER, TRACE_MARKER_TYPE_CHUNK_INSTR_COUNT, { 3 } },
         { TRACE_TYPE_MARKER, TRACE_MARKER_TYPE_TIMESTAMP, { 1002 } },
         { TRACE_TYPE_MARKER, TRACE_MARKER_TYPE_CPU_ID, { 2 } },
         { TRACE_TYPE_ENCODING, 8, { ENCODING_REGDEPS_ISA } },
         { TRACE_TYPE_INSTR, 3, { PC } },
+        { TRACE_TYPE_ENCODING, 16, { ENCODING_REGDEPS_ISA_vpmovmskb } },
+        { TRACE_TYPE_INSTR, 4, { PC_vpmovmskb } },
+        { TRACE_TYPE_ENCODING, 16, { ENCODING_REGDEPS_ISA_vmovdqu } },
+        { TRACE_TYPE_INSTR, 4, { PC_vmovdqu } },
         { TRACE_TYPE_FOOTER, 0, { 0 } },
     } };
     std::string expect =
         std::string(R"DELIM(           1           0:           3 <marker: version 3>
            2           0:           3 <marker: filetype 0x20e00>
            3           0:           3 <marker: cache line size 64>
-           4           0:           3 <marker: chunk instruction count 2>
+           4           0:           3 <marker: chunk instruction count 3>
            5           0:           3 <marker: timestamp 1002>
            6           0:           3 <marker: tid 3 on core 2>
            7           1:           3 ifetch       )DELIM") +
         std::string(R"DELIM(3 byte(s) @ 0x00007f6fdd3ec360 )DELIM") +
+        std::string(R"DELIM(00010011 move [8byte]       %rv4 -> %rv7
+)DELIM") +
+        std::string(R"DELIM(           7           1:           3)DELIM") +
+        std::string(
+            R"DELIM(                                             00060906
```

Comment:
I'm finding this quite difficult to read when it split over all these lines with the DELIM in there.  It is hard to figure out what this "006*" is supposed to be, sitting here by itself with tons of spaces all around.  I think it would be best to have one matching line per output line even if it's long.

Reply:
Initially I did that, but DR wouldn't build because of the 90 char per line limit.
I thought it was a clang-format issue, so I tried:
```
/* clang-format off */
{ code with lines over 90 chars}
/* clang-format on */
```

But it didn't work.
I'll investigate what I need to turn off to have one matching line per output.

Reply:
That `clang-format off` is how to disable clang-format.  There are also vera++ checks but I forget what they cover these days (most added before we had clang-format in place): I do see make/style_checks/scripts/rules/line_length_content.tcl and line_length_content is enabled.  Add this file to the exclusion list in make/style_checks/scripts/rules/line_length_content.tcl?

Reply:
Ah, yes, that should do the trick, thank you Derek!

Reply:
Not relevant to this PR anymore.